### PR TITLE
fix: Remove upcoming events from after event email text

### LINKED ISSUE #6623: After-Event Email with unrelated events
The after-event email includes the following:

"Thank You for participating in our event. We hope you enjoyed it. Please check the list of more upcoming events.
Here are the upcoming events: ..."

Please change this as follows.
* Only show "Thank You for participating in our event. We hope you enjoyed it. Please check out other upcoming events around you on eventyay.com"
* Take out the list of upcoming events and instead link to eventyay.com

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6623 

#### Short description of what this resolves:
Changes text of email

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal Check this

**Comment by iamareebjamal** [comment]
Read the complete issue again

**Comment by codedsun** [comment]
>Take out the list of upcoming events and instead link to eventyay.com

Currently they are linked to? as from the code the frontend_url is grabbed from settings.py. But can you explain me what they meant either the frontend url is wrong ? or the event id's are wrong?

File: app/api/helpers/system_mails.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
         'message': (
             u"Hi {email},<br/>" +
             u"Thank You for participating in our event. We hope you enjoyed it. "
-            u"Please check the list of more upcoming events. <br />" +
+            u"Please check out other upcoming events around you on eventyay.com <br />" +
```
Comment (kushthedude) [inline app/api/helpers/system_mails.py]: `Take out the list of upcoming events and instead link to eventyay.com` 
@codedsun

Comment (codedsun) [reply app/api/helpers/system_mails.py]: Please read my comment here 
https://github.com/fossasia/open-event-server/pull/6624#issuecomment-558815447

@kushthedude

Comment (kushthedude) [reply app/api/helpers/system_mails.py]: Remove the list of upcoming events which are being shown .

Comment (codedsun) [reply app/api/helpers/system_mails.py]: Omg, I thought take out means extraction -_-

**Comment by codedsun** [comment]
@kushthedude Review now!

File: app/api/helpers/system_mails.py [hunk 2]
```diff
@@ -63,8 +63,7 @@
         'message': (
             u"Hi {email},<br/>" +
             u"Thank You for participating in our event. We hope you enjoyed it. "
-            u"Please check the list of more upcoming events. <br />" +
-            u"Here are the upcoming events: {upcoming_events}. Get ready!! "
+            u"Please check out other upcoming events around you on eventyay.com <br />"
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py]: This repository is for open-event-server, not eventyay.com

Comment (codedsun) [reply app/api/helpers/system_mails.py]: have to use this via frontend_url? @iamareebjamal

**Comment by codedsun** [comment]
@iamareebjamal check now

File: app/api/helpers/scheduled_jobs.py [hunk 3]
```diff
@@ -29,13 +29,6 @@ def send_after_event_mail():
     from app import current_app as app
     with app.app_context():
         events = Event.query.filter_by(state='published', deleted_at=None).all()
-        upcoming_events = get_upcoming_events()
```
Comment (iamareebjamal) [inline app/api/helpers/scheduled_jobs.py]: If `get_upcoming_events` is not used anywhere else, comment on it saying `TODO: Unused function. Remove`

Comment (codedsun) [reply app/api/helpers/scheduled_jobs.py]: Done

**Comment by codedsun** [comment]
@iamareebjamal Check now

**Comment by codedsun** [comment]
thank you so much :)

**Pull request merged**